parm/use_cases/model_applications/medium_range/TCStat_SeriesAnalysis_fcstGFS_obsGFS_FeatureRelative_SeriesByInit.conf

```diff
@@ -1,70 +1,125 @@
+TC_PAIRS_OUTPUT_TEMPLATE = {date?fmt=%Y%m}/{basin?fmt=%s}q{date?fmt=%Y%m%d%H}.gfso.{cyclone?fmt=%s}
+
+TC_PAIRS_SKIP_IF_REFORMAT_EXISTS = yes
+TC_PAIRS_SKIP_IF_OUTPUT_EXISTS = yes
+
+
+TC_STAT_LOOKIN_DIR = {TC_PAIRS_OUTPUT_DIR}
+
+TC_STAT_OUTPUT_DIR = {EXTRACT_TILES_OUTPUT_DIR}
+TC_STAT_DUMP_ROW_TEMPLATE = {init?fmt=%Y%m%d_%H}/filter_{init?fmt=%Y%m%d_%H}.tcst
+
+
+EXTRACT_TILES_TC_STAT_INPUT_DIR = {TC_STAT_OUTPUT_DIR}
+EXTRACT_TILES_TC_STAT_INPUT_TEMPLATE = {TC_STAT_DUMP_ROW_TEMPLATE}
+
+FCST_EXTRACT_TILES_INPUT_DIR = {INPUT_BASE}/model_applications/medium_range/reduced_model_data
+FCST_EXTRACT_TILES_INPUT_TEMPLATE = {init?fmt=%Y%m%d}/gfs_4_{init?fmt=%Y%m%d}_{init?fmt=%H}00_{lead?fmt=%3H}.grb2
+
+OBS_EXTRACT_TILES_INPUT_DIR = {INPUT_BASE}/model_applications/medium_range/reduced_model_data
+OBS_EXTRACT_TILES_INPUT_TEMPLATE = {valid?fmt=%Y%m%d}/gfs_4_{valid?fmt=%Y%m%d}_{valid?fmt=%H}00_000.grb2
+
+EXTRACT_TILES_OUTPUT_DIR = {OUTPUT_BASE}/extract_tiles
+FCST_EXTRACT_TILES_OUTPUT_TEMPLATE = {init?fmt=%Y%m%d_%H}/{storm_id}/FCST_TILE_F{lead?fmt=%3H}_gfs_4_{init?fmt=%Y%m%d}_{init?fmt=%H}00_{lead?fmt=%3H}.nc
+OBS_EXTRACT_TILES_OUTPUT_TEMPLATE = {init?fmt=%Y%m%d_%H}/{storm_id}/OBS_TILE_F{lead?fmt=%3H}_gfs_4_{init?fmt=%Y%m%d}_{init?fmt=%H}00_{lead?fmt=%3H}.nc
+
+EXTRACT_TILES_SKIP_IF_OUTPUT_EXISTS = yes
+
+
+SERIES_ANALYSIS_TC_STAT_INPUT_DIR = {SERIES_ANALYSIS_OUTPUT_DIR}
+SERIES_ANALYSIS_TC_STAT_INPUT_TEMPLATE = {TC_STAT_DUMP_ROW_TEMPLATE}
+
+FCST_SERIES_ANALYSIS_INPUT_DIR = {EXTRACT_TILES_OUTPUT_DIR}
+FCST_SERIES_ANALYSIS_INPUT_TEMPLATE = {FCST_EXTRACT_TILES_OUTPUT_TEMPLATE}
+
+OBS_SERIES_ANALYSIS_INPUT_DIR = {EXTRACT_TILES_OUTPUT_DIR}
+OBS_SERIES_ANALYSIS_INPUT_TEMPLATE = {OBS_EXTRACT_TILES_OUTPUT_TEMPLATE}
+
+SERIES_ANALYSIS_OUTPUT_DIR = {OUTPUT_BASE}/series_analysis_init
+SERIES_ANALYSIS_OUTPUT_TEMPLATE = {init?fmt=%Y%m%d_%H}/{storm_id}/series_{fcst_name}_{fcst_level}.nc
+
+
+###
+# Field Info
+# https://metplus.readthedocs.io/en/latest/Users_Guide/systemconfiguration.html#field-info
+###
 
-# List of models to be used (white space or comma separated) eg: DSHP, LGEM, HWRF
-# If no models are listed, then process all models in the input file(s).
 MODEL = GFSO
 
-# List of storm ids of interest (space or comma separated) e.g.: AL112012, AL122012
-# If no storm ids are listed, then process all storm ids in the input file(s).
-TC_PAIRS_STORM_ID =
+BOTH_VAR1_NAME = TMP
+BOTH_VAR1_LEVELS = Z2
 
-# Basins (of origin/region).  Indicate with space or comma-separated list of regions, eg. AL: for North Atlantic,
-# WP: Western North Pacific, CP: Central North Pacific, SH: Southern Hemisphere, IO: North Indian Ocean, LS: Southern
-# Hemisphere
-TC_PAIRS_BASIN =
 
-# Cyclone, a space or comma-separated list of cyclone numbers. If left empty, all cyclones will be used.
-TC_PAIRS_CYCLONE =
+###
+# TCPairs Settings
+# https://metplus.readthedocs.io/en/latest/Users_Guide/wrappers.html#tcpairs
+###
+
+TC_PAIRS_INIT_INCLUDE =
```

Comment:
Should these options (e.g. TC_PAIRS_INIT_INCLUDE, TC_PAIRS_VALID_BEG, etc.) be removed if they're just being set to default (aka empty)? That seems to be more in line with how the other use cases are being handled.